Gemfile

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
+ruby '3.1.0'
 gem 'rspec'
+gem 'rubocop-gemfile', require: false
+gem 'simplecov', require: false, group: :test
```

Comment:
FYI, you're able to organize your gemfile groups like this as well:

```rb
group :test do
  gem 'simplecov', require: false
end
``` 
You can check this link for a little more information about that: [Bundler and Gemfile best practices](https://depfu.com/blog/2017/01/18/bundler-and-gemfile-best-practices#use-gemfile-groups)

Your change is good to go.